docs/modules/operation/pages/database-reports/database.adoc

```diff
@@ -73,157 +73,162 @@ In addition, create a new entry in the `$\{OPENNMS_HOME}/etc/jasper-reports.xml`
 
 To access performance data within reports, we created a custom Measurement Datasource that lets you query the Measurements API and process the returned data in your reports.
 Please refer to the link:http://docs.opennms.org/opennms/branches/develop/guide-development/guide-development.html#_measurements_api[official Measurements API documentation] on how to use the Measurements API.
 
-NOTE:  When using the Measurements Datasource within a report, an `HTTP` connection to the Measurements API is only established if the report is NOT running within {page-component-title}, e.g., when used with Jaspersoft Studio.
+NOTE:  When using the Measurements Datasource within a report, an HTTP connection to the Measurements API is established only if the report is not running within {page-component-title}; for example, when used with Jaspersoft Studio.
 
-To receive data from the Measurements API simply create a query as follows:
+To receive data from the Measurements API, create a query as follows:
 
-.Sample queryString to receive data from the `Measurements API`
+.Sample queryString to receive data from the Measurements API
 [source, xml]
 ------
 <query-request step="300000" start="$P{startDateTime}" end="$P{endDateTime}" maxrows="2000"> <1>
   <source aggregation="AVERAGE" label="IfInOctets" attribute="ifHCInOctets" transient="false" resourceId="node[$P\{nodeid}].interfaceSnmp[$P{interface}]"/>
   <source aggregation="AVERAGE" label="IfOutOctets" attribute="ifHCOutOctets" transient="false" resourceId="node[$P\{nodeid}].interfaceSnmp[$P{interface}]"/>
 </query-request>
 ------
-<1> The query language. In our case, `measurement`, but `JasperReports` supports a lot out of the box, such as `sql`, `xpath`, etc.
+<1> The query language. In our case, "measurement", but JasperReports supports a lot out of the box, such as sql, xpath, and so on.
 
 [[ga-database-reports-fields]]
 === Fields
 
-Each datasource should return a number of fields, which can be used in the report.
+Each data source should return a number of fields, which you can use in the report.
 The Measurement Datasource supports the following fields:
 
 [options="header"]
+[cols="1,1,3"]
 |===
 | Field name  | Field type         | Field description
-| `<label>`   | `java.lang.Double` | Each `Source` defined as `transient=false` can be used as a field.
-                                                        The name of the field is the `label`, e.g., `IfInOctets`
-| `timestamp` | `java.util.Date`   | The timestamp of the sample.
-| `step`      | `java.lang.Long`   | The `Step` size of the `Response`. Returns the same value for all rows.
-| `start`     | `java.lang.Long`   | The `Start` timestamp in milliseconds of the `Resopnse`. Returns the same value for all rows.
-| `end`       | `java.lang.Long`   | The `End` timestamp in milliseconds of the `Response`. Returns the same value for all rows.
+| <label>   | java.lang.Double | Each source defined as `transient=false` can be used as a field.
```

Comment:
Should this have the < > surrounding the name?

Reply:
nope. thanks!